safe-global/safe-wallet-web
# fix: recovery proposal icon in light.dark mode

## What it solves

Resolves inconsistent recovery proposal logo

## How this PR fixes it

A light/dark version of the proposal logo is used in accordance with dark mode. Separate icons were chosen as due to the complexity of the logo - our dynamic classes do not work well.

## How to test it

1. Ensuring recovery is enabled, open the Safe as a Guardian.
2. Observe the proposal recovery widget with a logo that looks "correct" in both light/dark mode.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/528ed30c-613b-41ba-8004-2ecc7ef5dbd1)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/2e99a806-0fd0-4fa1-b667-e1876ac77363)

## Checklist
